app/Controllers/CsvImportController.php

```
@@ -366,6 +372,19 @@ public function processChunk(Request $request, Response $response, \mysqli $db):
+                // Contributor roles are entities too (#237). CSV writes the
+                // retained legacy columns for round-trip compatibility, then
+                // resolves the same values into libri_autori in this transaction
+                // so post-migration imports never drift back to free text only.
+                $importData['authors_created'] += \App\Support\ContributorSync::linkLegacyValues(
+                    $db,
+                    $bookId,
+                    [
+                        'traduttore' => $parsedData['traduttore'] ?? null,
+                        'illustratore' => $parsedData['illustratore'] ?? null,
+                    ]
+                );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\ContributorSync' in method 'processChunk'. (undefined)

(StaticAccess)